services/calendars/Makefile

```diff
@@ -1,2 +1,3 @@
 calendars: $(GOVUK_ROOT_DIR)/calendars
 	$(COMPOSE_RUN) calendars-lite bundle
```

Comment:
I'm doing a PR to add an explicit 'build' step for the 'lite' service. Can you add one in here?

Reply:
https://github.com/alphagov/govuk-docker/pull/110

Reply:
Done